cmd/kubelet/app/options/options.go

```diff
@@ -316,6 +319,7 @@ func (f *KubeletFlags) AddFlags(mainfs *pflag.FlagSet) {
 	f.addOSFlags(fs)
 
 	fs.StringVar(&f.KubeletConfigFile, "config", f.KubeletConfigFile, "The Kubelet will load its initial configuration from this file. The path may be absolute or relative; relative paths start at the Kubelet's current working directory. Omit this flag to use the built-in default configuration values. Command-line flags override configuration from this file.")
+	fs.StringVar(&f.KubeletInstanceConfigFile, "instance-config", f.KubeletInstanceConfigFile, "The Kubelet will load its initial instance specific configuration from this file. The path may be absolute or relative; relative paths start at the Kubelet's current working directory. Omit this flag to use the built-in default configuration values. Command-line flags override configuration from this file.")
```

Comment:
the KEP for instance specific config had a proposal to use --config for both:
https://github.com/kubernetes/enhancements/pull/1439

Reply:
this of course means there has to be a separate instance specific Kind.